composer.json

```diff
@@ -79,7 +79,7 @@
         "nelmio/api-doc-bundle": "^2.13.4",
         "sensio/framework-extra-bundle": "^5.1",
         "sonata-project/admin-bundle": "^3.18.3",
-        "sonata-project/block-bundle": "^3.17",
+        "sonata-project/block-bundle": "^3.18",
```

Comment:
No adjustment here?

Reply:
It is require-dev section. Someone can use `--no-dev` so I change conflict too. Now obły 3.18.5 is possibile to use. Adjustment is not require here.

Reply:
You might want to reconsider because of the timeouts. Requiring `3.18.5` will reduce the problem space and could help